DfE.FindInformationAcademiesTrusts/Pages/Trusts/Ofsted/CurrentRatings.cshtml

```diff
@@ -8,7 +8,7 @@
 
 <section class="govuk-!-margin-top-8 app-table-container" data-testid="ofsted-current-ratings-table">
   <table class="govuk-table" data-module="moj-sortable-table">
-    <caption class="govuk-table__caption govuk-table__caption--m">Current ratings</caption>
+    <caption class="govuk-table__caption govuk-table__caption--m" data-testid="page-name">Current ratings</caption>
```

Comment:
On the other sub pages we used the data-testid "subpage-header", "page-name" is reserved for the h1 of the page

Reply:
Now updated to reflect the above